src/main/scala/com/cognite/sdk/scala/v1/containers/PrimitivePropType.scala

```diff
@@ -0,0 +1,49 @@
+// Copyright 2020 Cognite AS
+// SPDX-License-Identifier: Apache-2.0
+
+package com.cognite.sdk.scala.v1.containers
+
+import io.circe.{Decoder, Encoder}
+
+import java.util.Locale
+
+sealed abstract class PrimitivePropType extends Product with Serializable
+
+object PrimitivePropType {
```

Comment:
would be great if we can reuse those from [PropertyType](https://github.com/cognitedata/cognite-sdk-scala/blob/190e7042f6a3e7d96e0bcd5bcb2b3db178187921/src/main/scala/com/cognite/sdk/scala/v1/dataModelProperties.scala#L45).  Or it's better if they can be used for instances. @silvavelosa any thought on this?